zulip_botserver/zulip_botserver/server.py

```diff
@@ -28,6 +29,39 @@ def read_config_section(parser: configparser.ConfigParser, section: str) -> Dict
     }
     return section_info
 
+def read_config_from_env_vars(bot_name: Optional[str] = None) -> Dict[str, Dict[str, str]]:
+    bots_config = {}  # type: Dict[str, Dict[str, str]]
+    env_config = OrderedDict()   # type: Dict[str, Dict[str, str]]
+
+    # Check for presence of environment variables
+    # otherwise throws appropriate errors
+    if os.environ.get('ZULIP_BOTSERVER_CONFIG') is None:
+        raise OSError("Could not read environment varibale 'ZULIP_BOTS': Variable not set.")
+
+    # The bots emails keys sites and tokens
+    # are stored in their respective lists
+    env_config = json.loads(str(os.environ.get('ZULIP_BOTSERVER_CONFIG')), object_pairs_hook=OrderedDict)
+
+    # If bot name is provided we only use
+    # the provided name and return it's config
```

Comment:
`return its config`, not `return it's config`, in this case.

I also notice your comment lines are wrapped fairly short; you should wrap them at 80-ish characters.